iothub/service/src/FeedbackStatusCode.cs

```diff
@@ -12,10 +12,12 @@ namespace Microsoft.Azure.Devices
     [JsonConverter(typeof(StringEnumConverter))]
     public enum FeedbackStatusCode
     {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Why not document?

Reply:
The values were very self explanatory so I chose to not document. The values are Success, Expired, Rejected etc... It felt redundant to not add any extra information.

Reply:
Even if it seems obvious, I'd prefer we go one step further and try to describe them in plain English. You never know, your PR reviewers may have ideas for improving the descriptions or adding more context.

Reply:
what is "DeliveryCountExceeded" for?

Reply:
Sure. All up for that! I don't know what I would add myself. If someone has ideas, I'll add those as comments.

Reply:
And I did look up for all of this in the IotHub repo too but they also don't have any comments for most of the things :( I had to scramble all of this from multiple places and looking at code. For this one specifically I don't have much so if someone can add information, I can add it.

Reply:
Perhaps the swagger or the web documentation could be a good source of descriptions?

Reply:
I already looked at the web documentation and found nothing. I looked in the swagger now and did not find anything. If you find something, can you paste a link here?

Reply:
Okay, let's try:

- Success: `The message sent successfully.`
- Expired: Not sure on this one.
- DeliveryCountExceeded: `Failed to send due to exceeding quota.`
- Rejected: `The message failed to send due to being rejected.`
- Purged: `The message was purged in the queue before being accepted.`

Reply:
As discussed, i'll remove the compiler directive for now. Once we get more information, I will send another PR.